dspace/config/modules/storage.cfg

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------#
+#-----------------STORAGE CONFIGURATIONS------------------------#
+#---------------------------------------------------------------#
+# Configuration properties used by the bitstore.xml config file #
+#                                                               #
+#---------------------------------------------------------------#
+
+# Use the localStore or the s3Store implementation
+assetstore.storename.0 = localStore
+
+# For using a relative path (xx/xx/xx/xxx...) set to true, default it false
+assetstore.s3.useRelativePath = false
+
+## Assetstore S3 configuration, only used if the above configuration
+## is set to s3Store
+
+# S3 bucket name to store assets in, default would generate a bucket
+# based on the dspace host name
+assetstore.s3.bucketName =
+# Subfolder to organize assets within the bucket, in case this bucket
+# is shared. Optional, default is root level of bucket
+assetstore.s3.subfolder =
+
+# please do not use these in production but rely on the aws credentials
+# discovery mechanism to configure them (ENV VAR, EC2 Iam role, etc.)
+assetstore.s3.awsAccessKey =
+assetstore.s3.awsSecretKey =
+# to force the use of a specific region when credentials are provided
+# in this configuratin file. If credentials are left empty this prop
+# is ignored
```

Comment:
I don't think this description is accurate.  From the code, it looks like `awsRegionName` is set to the AWS `DEFAULT_REGION` (which I think is `us-east-1`?) if it's not specified.

Reply:
Yes, the `awsRegionName` is configured with the `DEFAULT_REGION` if the property is left blank.
But the comment
> to force the use of a specific region when credentials are provided
> in this configuratin file. **If credentials are left empty this prop
>  is ignored**
assures that if credentials are empty the region is ignored, because we will use the default credential manager of AWS

Reply:
@vins01-4science : I'd recommend describing that better then.  I do not think the current text is clear.  Maybe say something like:

`If the credentials are left empty, then this setting is ignored and the default AWS region will be used.`